### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Fostrankill**
Updated Carousel (Swiper)

### CODE REVIEW

**Comment by LoicBrassart** [comment]
Missing the SM review

File: frontend/src/App.css [hunk 1]
```diff
@@ -6,18 +6,17 @@
...
   display: flex;
   justify-content: center;
   align-items: center;
   z-index: 15;
   grid-area: 2 / 1 / 5 / 4;
-}
+} 
 
 .layout {
-  display: grid;
```
Comment (AlexisFLT) [inline frontend/src/App.css]: Display Layout has been removed ? need to restore it

**Review by AlexisFLT — CHANGES_REQUESTED** [review]
display grid to fix : .layout

**Review by AlexisFLT — COMMENTED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
Mising SM validation :smirk:

**Review by AlexisFLT — DISMISSED** [review]
Ok for me

File: frontend/package.json [hunk 2]
```diff
@@ -12,7 +12,9 @@
   "dependencies": {
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "styled-components": "^5.3.6",
```
Comment (LoicBrassart) [inline frontend/package.json]: Nein! Votre site est stylé en css vanilla, pas besoin de ce package

File: frontend/src/components/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { EffectCoverflow } from "swiper";
+/* eslint-disable import/no-unresolved */
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/effect-coverflow";
+// eslint-disable-next-line import/no-unresolved
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: /* eslint-enable import/no-unresolved */

File: frontend/src/components/Carousel/Carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import Card from "@components/Cards/Card/Index";
+
+import SCarousel from "./style";
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: No styled-components!

File: package.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
   "license": "ISC",
   "devDependencies": {
     "concurrently": "^7.0.0",
+    "eslint": "^8.26.0",
```
Comment (LoicBrassart) [inline package.json]: Wait, what ? Why ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope, please keep your coding conventions for this project

File: frontend/src/components/Carousel/Carousel.jsx (hunk 3 above)
Comment (AlexisFLT) [reply frontend/src/components/Carousel/Carousel.jsx]: Romain me dit que c'est le quickfix obligatoire au .swipper

File: package.json (hunk 5 above)
Comment (AlexisFLT) [reply package.json]: Je comprend pas

**Review by AlexisFLT — APPROVED** [review]
Now i think it's ok

**Review by LoicBrassart — APPROVED** [review]
At last! ;-)

**Pull request merged**